Content/src/Server/ServerSaturn.fs

```diff
@@ -19,10 +19,6 @@ let port = 8085us
 
 let getInitCounter () : Task<Counter> = task { return 42 }
 
-let browserRouter = scope {
```

Comment:
It's still used, when `Remoting` is requested

Reply:
Doh. Good spot - will fix.